app/assets/stylesheets/profiles.scss

```diff
@@ -6,43 +6,53 @@
 /* profile view */
 .profile-overview {
   margin-top: 10px;
-  margin-right: 20px;
-  border: 5px solid white;
-  border-radius: 5px;
+  border: 3px solid white;
+  border-radius: 20px;
   background-color: black;
   color: white;
```

Comment:
This branch is way behind master.  I don't think I touched anything with search, but I will try to fix this while merging